xwiki-platform-core/xwiki-platform-filter/xwiki-platform-filter-instance/xwiki-platform-filter-instance-document/src/main/java/org/xwiki/filter/instance/output/EntityInstanceOutputProperties.java

```diff
@@ -59,7 +59,7 @@ public class EntityInstanceOutputProperties extends InstanceOutputProperties
      * @return the comment to use when saving users
```

Comment:
This comment isn't consistent with the property's description.

Reply:
Done.